src/main/java/gov/usgs/earthquake/indexer/ExtentIndexerListener.java

```diff
@@ -56,7 +59,17 @@ public void processProduct(ProductSummary product) throws Exception {
     super.processProduct(product);
     ExtentSummary extent = new ExtentSummary(product);
 
-    ((ExtentIndex) productIndex).addExtentSummary(extent);
+    if (extent.isValid()) {
+      LOGGER.log(Level.FINE, "[" + getName() + "] adding product "
+          + product.getIndexId()
```

Comment:
Is the product id available, instead of the index id...

Reply:
used ProductId instead of the index Id